abstutil/Cargo.toml

```diff
@@ -10,6 +10,7 @@ instant = "0.1.7"
 itertools = "0.9.0"
 lazy_static = "1.4.0"
 log = { version = "0.4.11", features=["std"] }
+env_logger = { version = "0.8.2" }
```

Comment:
nit: Please sort

Reply:
Sorry that I always forget to do this. Fixed!